test/extensions/filters/http/ext_proc/filter_test.cc

```diff
@@ -1207,6 +1278,265 @@ TEST_F(HttpFilterTest, PostFastAndBigRequestPartialBuffering) {
   filter_->onDestroy();
 }
 
+// Streaming sends body with small chunks.
+TEST_F(HttpFilterTest, StreamingDataSmallChunk) {
+  initialize(R"EOF(
```

Comment:
You may want to isolate the common code to make these tests smaller.

Reply:
done